docs/model_developers/handlers.rst

```diff
@@ -134,18 +139,44 @@ method. e.g.:
             for key, value in ...:
                 ctx.set_fact(fact_id=key, value=value, expires=True)
 
-By default, facts expire when they are not being refreshed or updated for a certain time, controlled by the
+
+By default, facts expire when they have not been refreshed or updated for a certain time, controlled by the
 :inmanta.config:option:`server.fact-expire` config option. Querying for an expired fact will force the
 agent to refresh it first.
-Expired facts are also refreshed periodically. This time interval is controlled by the
-:inmanta.config:option:`server.fact-renew` config option.
-
 
 When reporting a fact, setting the ``expires`` parameter to ``False`` will ensure that this fact never expires. This
 is useful to take some load off the agent when working with facts whose values never change. On the other hand, when
 working with facts whose values are subject to change, setting the ``expires`` parameter to ``True`` will ensure
 they are periodically refreshed.
 
+---------
+
+Facts are automatically pulled periodically (this time interval is controlled by the
```

Comment:
They are only pulled if they are about to expire or if a model requested them and they where not present.